db/migrate/20230509162350_create_departments.rb

```diff
@@ -0,0 +1,14 @@
+class CreateDepartments < ActiveRecord::Migration[7.0]
+  def up
+    create_table :departments do |t|
+      t.string :name
+      t.string :abbreviation
+      t.integer :manager_id
```

Comment:
it should be a reference

Reply:
I'm not sure about this attribure. Should we keep managers in a separate model or we can take employee from Employee model as a manager?

Reply:
or it should be just a record in Department model without any ID and info?